tools/mo/unit_tests/mo/ops/scatter_test.py

```diff
@@ -101,3 +100,70 @@ def test_scatterelements_value_infer(self, data, indices, updates, axis, ref_res
 
         res_output_value = scatter_el_node.out_node().value
         self.assertTrue(np.array_equal(ref_res, res_output_value))
+
+
+@generator
+class ScatterUpdateInferTest(unittest.TestCase):
+    @generate(*[
+        ([[0.0, 0.0, 0.0],
+          [0.0, 0.0, 0.0],
+          [0.0, 0.0, 0.0]],
+         [[1, 2]],
+         [[[1.0, 1.1, 1.2],
+           [2.0, 2.1, 2.2]]],
+         0,
+         [[0.0, 0.0, 0.0],
+          [1.0, 1.1, 1.2],
+          [2.0, 2.1, 2.2]]),
+
+        # negative axis
```

Comment:
can you please add a cases with the following shapes:
```
data_shape = [2, 3, 3]
indices_shape = [3, 2]
updates shape = [2, 3, 2, 3]
axis = 1
```

Reply:
Done